src/site/includes/image_detail_mobile_responsive.drupal.liquid

```diff
@@ -0,0 +1,25 @@
+<div classname="vads-u-display--flex vads-u-margin-bottom--2">
+    {% assign alt = mediaImage.entity.image.alt %}
+    <div class="vads-l-grid-container vads-u-padding-x--0 vads-u-margin-y--2 ">
+        <div class="vads-l-row">
+            <dt class="vads-l-col--12 medium-screen:vads-u-display--none vads-u-margin-bottom--2">
+                <{{image_title_tag}} class="{{image_title_classes }} medium-screen:vads-u-margin-top--0">{{ mediaImage.entity.entityLabel }}</{{image_title_tag}}>
+            </dt>
+            <dd class=".vads-l-col--12 medium-screen:vads-l-col--2">
+                <picture>
+                    {%if mediaImage.entity.image.url == empty %}
+                        <img class="vads-u-min-height--viewport vads-u-background-color--gray-lightest vads-u-display--block vads-u-border--1px vads-u-border-color--gray-light" alt="{{ alt }}" height="{{ mediaImage.entity.image.height }}" width="{{ mediaImage.entity.image.width }}" />
```

Comment:
So is this just a blank image then?

Reply:
yup! this is for displaying the grey background when the alt text is displayed